aten/src/ATen/native/mps/operations/ReduceOps.mm

```diff
@@ -40,7 +40,7 @@ void set_apparent_shapes(NSMutableArray<NSNumber*>*& apparent_out_shape,
                          int64_t num_reduce_dims,
                          int64_t num_output_dims,
                          IntArrayRef& input_shape,
-                         NSMutableArray<NSNumber*>*& axes) {
+                         NSArray<NSNumber*>* axes) {
```

Comment:
It appears that the set_apparent_shapes function only requires read-only access to the axes argument, so there is no need for it to be an NSMutableArray.